### PR TITLE
fix(CX-1529): unable to edit credit card

### PR DESCRIPTION
**Pull request by edez1t**
The type of this PR is: **Bugfix**

This PR resolves [CX-1529]

### Description
This PR fixes card input on android

https://user-images.githubusercontent.com/73339470/120336556-2481d380-c2fb-11eb-84f9-1699d43376f0.mov

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/master/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [x] I have added a [CHANGELOG.yml](/CHANGELOG.yml) entry or my changes do not require one.

[CX-434]: https://artsyproduct.atlassian.net/browse/CX-434
[CX-1529]: https://artsyproduct.atlassian.net/browse/CX-1529

### CODE REVIEW

File: patches/react-native-credit-card-input+0.4.1.patch [hunk 1]
```diff
@@ -35,9 +35,21 @@ index dde3aec..aec1f45 100644
...
+   StyleSheet,
+   Image,
+-  LayoutAnimation,
+   TouchableOpacity,
+   TextInput,
+ } from "react-native";
++import { Transition, Transitioning } from 'react-native-reanimated';
+ 
+ import Icons from "./Icons";
+ import CCInput from "./CCInput";
```
Comment (MounirDhahri) [inline patches/react-native-credit-card-input+0.4.1.patch:52]: **issue(blocker):** Do you mind if you add a comment about the changes you made in order for us to be able to remove this patch in the future. Also can you add it here https://github.com/artsy/eigen/blob/master/HACKS.md

**Review by MounirDhahri — CHANGES_REQUESTED** [review]

File: patches/react-native-credit-card-input+0.4.1.patch (hunk 1 above)
Comment (edez1t) [reply patches/react-native-credit-card-input+0.4.1.patch]: done

**Comment by MounirDhahri** [comment]
I think you forgot to add a short description of your changes and when we remove the patch to https://github.com/artsy/eigen/blob/master/HACKS.md

File: HACKS.md [hunk 2]
```diff
@@ -39,113 +43,142 @@ After a few months we should be safe to return to the old name if we want. If we
...
 We can remove these hacks once we switch to Stripe's forthcoming official react-native library.
 
 #### Explanation/Context:
+
 These are fairly superficial styling hacks for
+
 - focused/error border states
 - shrinking the icon size to work nicely with our inputs
 - aligning inner inputs nicely
+- icon animation to work properly on android
```
Comment (edez1t) [inline HACKS.md:170]: Should've saved this without formatting

**Review by brainbicycle — APPROVED** [review]
Nice fix! I'll let Mounir take another look but looks good to me.

**Pull request merged**